app/assets/stylesheets/common/_criteria_filter.scss

```diff
@@ -0,0 +1,44 @@
+@import 'constants';
+
+.criteria-filter {
+  box-shadow: 0 0 2px 0.5px $line;
```

Comment:
I'm not a fan of the box shadow. You can, however, put up a vertical border between the criteria filter and everything else